charts/seed-bootstrap/dashboards/webhook-details.json

```diff
@@ -0,0 +1,462 @@
+        "error": null,
+        "hide": 0,
+        "includeAll": false,
+        "label": null,
+        "multi": false,
+        "name": "job",
+        "options": [],
+        "query": {
+          "query": "label_values(controller_runtime_webhook_requests_total{}, job)",
+          "refId": "StandardVariableQuery"
+        },
+        "refresh": 2,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 0,
+        "tagValuesQuery": "",
+        "tags": [],
+        "tagsQuery": "",
+        "type": "query",
+        "useTags": false
+      },
+      {
+        "allValue": null,
+        "current": {
+          "selected": false,
+          "text": "/networkpolicy",
+          "value": "/networkpolicy"
+        },
+        "datasource": "${datasource}",
+        "definition": "label_values(controller_runtime_webhook_requests_total{job=\"$job\"},webhook)",
+        "description": null,
+        "error": null,
+        "hide": 0,
+        "includeAll": false,
+        "label": null,
+        "multi": false,
+        "name": "webhook",
+        "options": [],
+        "query": {
+          "query": "label_values(controller_runtime_webhook_requests_total{job=\"$job\"},webhook)",
+          "refId": "StandardVariableQuery"
+        },
+        "refresh": 2,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 0,
+        "tagValuesQuery": "",
+        "tags": [],
+        "tagsQuery": "",
+        "type": "query",
+        "useTags": false
+      },
+      {
+        "allValue": null,
+        "current": {
+          "selected": true,
+          "text": [
+            "All"
+          ],
+          "value": [
+            "$__all"
+          ]
+        },
+        "datasource": "${datasource}",
+        "definition": "label_values(controller_runtime_webhook_requests_total{job=\"$job\",webhook=\"$webhook\"},code)",
+        "description": null,
+        "error": null,
+        "hide": 0,
+        "includeAll": true,
+        "label": null,
+        "multi": true,
+        "name": "code",
+        "options": [],
+        "query": {
+          "query": "label_values(controller_runtime_webhook_requests_total{job=\"$job\",webhook=\"$webhook\"},code)",
+          "refId": "StandardVariableQuery"
+        },
+        "refresh": 2,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 0,
+        "tagValuesQuery": "",
+        "tags": [],
+        "tagsQuery": "",
+        "type": "query",
+        "useTags": false
+      },
+      {
+        "allValue": null,
+        "current": {
+          "selected": false,
+          "text": "All",
+          "value": "$__all"
+        },
+        "datasource": "${datasource}",
+        "definition": "label_values(controller_runtime_webhook_requests_total{job=\"$job\",webhook=\"$webhook\"},pod)",
+        "description": null,
+        "error": null,
+        "hide": 0,
+        "includeAll": true,
+        "label": null,
+        "multi": true,
+        "name": "pod",
```

Comment:
Can you move the `pod` variable between `job` and `webhook` for consistency with the controller details dashboard?

Reply:
What do you mean? pod is after `job` and `controller` in controller-details dashboard.

Reply:
Ah right, sorry for the confusion. I would go for the following order
`datasource`, `job`, `pod`, `controller/webhook`, (`code`)
That's basically the intuitive hierarchical order.

Reply:
Addressed in https://github.com/gardener/gardener/pull/7180/commits/8366983488f23574e11c52d772b55405c54ca247